addons/csi/kubevirt/csi-driver-operator.yaml

```diff
@@ -396,23 +396,21 @@ spec:
   storageClasses:
 {{- range  .Cluster.KubeVirtInfraStorageClasses }}
    - infraStorageClassName: {{ .Name }}
-{{- if .IsDefaultClass }}
      isDefaultClass: {{ .IsDefaultClass }}
-{{- end }}
```

Comment:
We should still retain this check IMO since IsDefaultClass is a boolean pointer. This would result in `isDefaultClass: nil` or `isDefaultClass: <nil>` if IsDefaultClass is not set.

Reply:
The thing is, I actually added that in a previous PR but then when I set it to false the field will be dropped because of the omitempty thing leading to the csi driver failure so I took it out 😅

Reply:
I would also feel more comfortable with a check, but since it's been this way before the changes in #13874 (see e.g. https://github.com/kubermatic/kubermatic/blob/v2.26.0/addons/csi/kubevirt/csi-driver-operator.yaml#L372), I guess we are just retaining the status quo and could iterate after the patch release.